src/plotter.py

```diff
@@ -274,6 +290,46 @@ def create_nan_graph(self, min_date: str, max_date: str):
             nan_dates = gauge_data[str(gauge)].index[gauge_data[str(gauge)].apply(np.isnan)].strftime("%Y-%m-%d")
 
             for date in nan_dates:
-                nan_graph.add_node(node_for_adding=(gauge, date))
+                nan_graph.add_node(node_for_adding=(gauge, date), color="green")
 
         return nan_graph
+
+    def get_subgraph_by_level_group(self, directed_graph: nx.DiGraph):
```

Comment:
Annotate return value, add docstring.

Reply:
Szerintem ezt a metódust első próbálkozásaim között írtam, amikor még nem tudtam, hogy van olyan beépített függvény, hogy `nx.is_weakly_connected()`. Ahogy elnézem, valami hasonlót próbáltam írni kézzel. Ez sehol máshol nincs használva a repo-ban (szerencsére, mert nem is működik), úgyhogy ezt törlöm inkább.